db/main.py

```diff
@@ -0,0 +1,68 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+def root() -> dict:
+    return {"message": "Hello form Library"}
+
+
+@app.get("/books/", response_model=list[schemas.Book])
+def read_books(db: Session = Depends(get_db)):
+    return crud.get_all_books(db=db)
+
+
+@app.post("/books/", response_model=schemas.Book)
+def create_book(
+    book: schemas.BookCreate,
+    db: Session = Depends(get_db),
+):
+    db_book = crud.get_book_by_title(db=db, title=book.title)
+
+    if db_book:
+        raise HTTPException(
+            status_code=400,
+            detail="Such title for Book already exists"
+        )
+
+    return crud.create_book(db=db, book=book)
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_cheese(
```

Comment:
The function name `read_cheese` seems incorrect. It should be `read_authors` to match the endpoint's purpose of retrieving authors.